samples/Ping/Ping/Ping.csproj

```diff
@@ -34,9 +34,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
@tavitruman any ideas on the compatibility of json.net 10.0? If there's improvement to the system we may consider an update. However, compatibility with older codebase will be dropped, as AssemblyBindingRedirection only works for mapping calls to an older assembly to a newer one but not the reverse.

Reply:
@yatli Yikes there are some compatibility issues with the latest version of json.net 10.0 so I am revert back to the prior release.